### PR TITLE
feat: add tabs to global html

### PR DESCRIPTION
**Pull request by pregno**
Normal state
Screenshot 2024-09-26 at 17 27 33

Focused state
Screenshot 2024-09-26 at 17 27 41

Hover state
Screenshot 2024-09-26 at 17 27 49

### CODE REVIEW

File: packages/html/ds/src/tabs/tabs.html
Comment (BogdanDanilescu) [inline packages/html/ds/src/tabs/tabs.html:1]: I would separate the macros into their own files just to have readability and keep the consistency compared to other components.

File: packages/html/ds/scripts/build-properties.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ function getSchemaProperties(schema) {
           description,
           type: toType(zodType),
           required,
-          values: zodEnums ? zodEnums : undefined,
+          values: zodEnums || undefined,
```
Comment (Thurfjell) [inline packages/html/ds/scripts/build-properties.ts:69]: Can `(value as any)._def.values` be 0, "" or null ? Otherwise why is the || needed?

**Review by Thurfjell — APPROVED** [review]

**Review by hamza14khan — APPROVED** [review]

**Comment by hamza14khan** [comment]
Would be great if you can add Screenshot/GIFs for tabs.

**Review by hamza14khan — CHANGES_REQUESTED** [review]
Looks good, can you please also update doc site and examples?

File: packages/html/ds/scripts/build-properties.ts (hunk 1 above)
Comment (pregno) [reply packages/html/ds/scripts/build-properties.ts:69]: good point

**Review by hamza14khan — APPROVED** [review]
Thanks for taking care of the comments!

**Pull request merged**